AGM_Interaction/config.cpp

```diff
@@ -186,6 +187,7 @@ class AGM_Core_Options {
 
 class AGM_Parameters {
   AGM_Modifier = 0;
+  AGM_enableTeamManagement = 1;
```

Comment:
You need to tell commy :smile:
https://github.com/KoffeinFlummi/AGM/commit/07caa59ebd19073c0b46427369bd7d764d8d2cea
But I can change this.